db/migrate/20230406181952_create_clubhouses.rb

```diff
@@ -0,0 +1,10 @@
+class CreateClubhouses < ActiveRecord::Migration[7.0]
+  def change
+    create_table :clubhouses do |t|
+      t.text :title, null: false
+      t.text :description
```

Comment:
use `t.string`.  `t.text` is for very long texts, like articles and such

Reply:
done!